smoke-tests/designer/data/componentFieldData.js

```diff
@@ -35,7 +35,7 @@ module.exports = {
           You can see the results as soon as the MOT centre has recorded the test result.\n
           You’ll need the 11-digit number from the vehicle’s log book (V5C) to see the test location.`,
   },
-  yesNo: {
+  yesno: {
```

Comment:
this seems less consistent

Reply:
@endamccormack an unfortunate change I had to make as it appears in the create component section as `YesNo` rather then `Yes No` so the only option was to change the case or come up with a special function that only gets used for one component.